### DISCUSSION TITLE
Whattomine is listed as a pool in RBM  Is there a pool associated with Whattomine?

### DISCUSSION THREAD

**Discussion by smyers63** [RainbowMiner/RainbowMiner#2321] [Q&A] [answered]
I haven't been able to find any mining pool related to Whattomine.  What does having it enabled or disabled in the pool list do for RBM?  Is it just a holdover from something that doesn't exist now?

If there's still something that RBM will do when that's enabled, what is it and how should it be configured to use it?

**Comment by RainbowMiner** [comment, marked as answer]
The Whattomine "pool" module is a drop-in replacement for all profitability calculations of pools, that don't offer enough API data to calculate the profit right away.

**Comment by smyers63** [comment]
Is there any difference between having it enabled and disabled in Setup?  I'm guessing there isn't.

![RBM-whattomine-enable1](https://user-images.githubusercontent.com/95280323/225800362-a898784a-14dd-46f2-aac0-3df39e722339.png)
![RBM-whattomine-enable2](https://user-images.githubusercontent.com/95280323/225800380-53a15122-c188-4790-b90f-0cf1f94485c1.png)

**Comment by RainbowMiner** [comment]
No, there isn't - it's an always on module.
